manifest.bioimage.io.yaml

```diff
@@ -19,7 +19,7 @@ config:
 
 application:
     - id: Ilastik
```

Comment:
if we want to change the id to `ilastik` here, we should do it now. As new models will create links to this id.